src/Connector.SqlServer/Connector/SqlServerConnector.cs

```diff
@@ -32,7 +32,12 @@ public class SqlServerConnector : SqlConnectorBase<SqlServerConnector, SqlConnec
         private readonly int _bulkInsertThreshold;
         private readonly bool _bulkSupported;
         private readonly bool _syncEdgesTable;
-        private readonly IList<string> _defaultKeyFields = new List<string> { "OriginEntityCode" };
+
+        private readonly IList<(string name, bool isUnique)> _defaultIndexFields = new List<(string, bool)>
```

Comment:
Could we make this IList<(string[] columns, bool isUnique)> instead so if we in the future would have multi column indexes we don't need to refactor everything again.